app/Platform/Actions/UpdatePlayerGameMetrics.php

```diff
@@ -61,8 +62,8 @@ public function execute(PlayerGame $playerGame, bool $silent = false): void
             ? $playerGame->created_at
             : $startedAt;
 
-        $lastPlayedAt = $playerAchievements->pluck('unlocked_at')
-            ->merge($playerAchievementsHardcore->pluck('unlocked_hardcore_at'))
+        $lastPlayedAt = $playerAchievements->whereNull('unlocker_id')->pluck('unlocked_at')
+            ->merge($playerAchievementsHardcore->whereNull('unlocker_id')->pluck('unlocked_hardcore_at'))
```

Comment:
This prevents the `last_played_at` from including manual unlock timestamps, which indirectly fixes the issue where manually unlocking an achievement moves the game up on the Recently Played games list (https://discord.com/channels/476211979464343552/1026595325038833725/1166399147063115877).